Assignment-10-August-22/userprofile/src/containers/Pages/Title.js

```diff
@@ -0,0 +1,6 @@
+const setDocumentTitle = (title) => {
```

Comment:
this is a utility function. create a utils folder under pages and move this file there and make file name with camelCase letter or small letters if its not a component.